### DISCUSSION TITLE
What is master-exportshapes branch?

### DISCUSSION THREAD

**Discussion by jsbien** [rmast/djvulibre#3] [General]
Is this somehow relevant for me?

**Comment by rmast** [comment]
That was the branch without the stripped revealshapes, exportshapes needs mysql or mariadb on bookworm.

I'm composing a new minimal branch/repo that will just link libdjvulibre.so as a shared library

**Comment by rmast** [comment]
By the way, exportshapes is really slow for some reason. One of the potential reasons is it trims the white border of some image.
